### PR TITLE
Fix warnings revealed by v0.14.1 PS release

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

Self-explanatory

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a link to this PR and your username
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation in the README and/or documentation directory
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by JordanMartinez** [comment]
I'm not sure how these warnings should be handled. I'm assuming we should export them?

```
[UnusedDeclaration] .spago/parsing/v6.0.0/src/Text/Parsing/Parser/Indent.purs:207:1

  207  indentBrackets p = withPos $ pure identity <-/> symbol "[" <+/> p <-/> symbol "]"
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  
  Declaration indentBrackets was not used, and is not exported.

[UnusedDeclaration] .spago/parsing/v6.0.0/src/Text/Parsing/Parser/Indent.purs:211:1

  211  indentAngles p = withPos $ pure identity <-/> symbol "<" <+/> p <-/> symbol ">"
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  
  Declaration indentAngles was not used, and is not exported.

[UnusedDeclaration] .spago/parsing/v6.0.0/src/Text/Parsing/Parser/Indent.purs:215:1

  215  indentBraces p = withPos $ pure identity <-/> symbol "{" <+/> p <-/> symbol "}"
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  
  Declaration indentBraces was not used, and is not exported.

[UnusedDeclaration] .spago/parsing/v6.0.0/src/Text/Parsing/Parser/Indent.purs:219:1

  219  indentParens p = withPos $ pure identity <-/> symbol "(" <+/> p <-/> symbol ")"
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  
  Declaration indentParens was not used, and is not exported.
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ New features:
 Bugfixes:
 
 Other improvements:
+- Fix warnings found by v0.14.1 PS release (#112)
+- Add dependencies whoes modules are used directly in source code (#112)
```
Comment (thomashoneyman) [inline CHANGELOG.md:15]:
```suggestion
- Removed unused names found by v0.14.1 PureScript release (#112 by @JordanMartinez)
- Installed dependencies directly imported into source code that were previously installed transitively (#112 by @JordanMartinez)
```

**Comment by thomashoneyman** [comment]
@JordanMartinez I'm surprised those warnings aren't showing up in the CI build; shouldn't they cause it to fail if the compiler is warning on them?

Otherwise, we can decide whether we want to export those functions or remove them.

**Comment by JordanMartinez** [comment]
@thomashoneyman CI isn't failing because [it's still using `v0.14.0`](https://github.com/purescript-contrib/purescript-parsing/runs/2376832559#step:3:12). Once we update it to v0.14.1, it will fail.

File: CHANGELOG.md (hunk 1 above)
Comment (JordanMartinez) [reply CHANGELOG.md]: Fixed in latest commit

**Comment by JordanMartinez** [comment]
Latest commit exports those 4 unused/unexported functions.

**Review by thomashoneyman — APPROVED** [review]

**Pull request merged**